StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -341,6 +363,61 @@ public static void HandleEventFieldDeclaration(SyntaxNodeAnalysisContext context
                     }
                 }
             }
+
+            private static string[] GetDocumentedParameters(SyntaxNodeAnalysisContext context, RecordDeclarationSyntaxWrapper typeDeclarationSyntax, out bool hasInheritdoc)
+            {
+                hasInheritdoc = false;
+
+                var documentation = context.Node.GetDocumentationCommentTriviaSyntax();
+                if (documentation == null)
+                {
+                    return EmptyArray<string>.Instance;
+                }
+
+                if (documentation.Content.GetFirstXmlElement(XmlCommentHelper.InheritdocXmlTag) != null)
+                {
+                    hasInheritdoc = true;
+                    return EmptyArray<string>.Instance;
+                }
+
+                var hasIncludedDocumentation =
+                    documentation.Content.GetFirstXmlElement(XmlCommentHelper.IncludeXmlTag) != null;
+
+                if (hasIncludedDocumentation)
+                {
+                    var declaredSymbol = context.SemanticModel.GetDeclaredSymbol(typeDeclarationSyntax, context.CancellationToken);
+                    var rawDocumentation = declaredSymbol?.GetDocumentationCommentXml(expandIncludes: true, cancellationToken: context.CancellationToken);
+                    var includedDocumentation = XElement.Parse(rawDocumentation ?? "<doc></doc>", LoadOptions.None);
+
+                    if (includedDocumentation.Nodes().OfType<XElement>().Any(element => element.Name == XmlCommentHelper.InheritdocXmlTag))
+                    {
+                        hasInheritdoc = true;
+                        return EmptyArray<string>.Instance;
+                    }
+
+                    IEnumerable<XElement> paramElements = includedDocumentation.Nodes()
+                        .OfType<XElement>()
+                        .Where(x => x.Name == XmlCommentHelper.ParamXmlTag);
+
+                    return paramElements
+                        .SelectMany(x => x.Attributes().Where(y => y.Name == XmlCommentHelper.NameArgumentName))
+                        .Select(x => x.Value)
+                        .ToArray();
+                }
+                else
+                {
+                    IEnumerable<XmlNodeSyntax> xmlNodes = documentation.Content.GetXmlElements(XmlCommentHelper.ParamXmlTag);
+                    return xmlNodes.Select(XmlCommentHelper.GetFirstAttributeOrDefault<XmlNameAttributeSyntax>)
+                        .Where(x => x != null)
+                        .Select(x => x.Identifier.Identifier.ValueText)
+                        .ToArray();
+                }
+            }
+
+            private static class EmptyArray<T>
```

Comment:
I would move it to the Helpers folder so it can be used in other places as well if needed

Reply:
I did a bit of refactoring and was no longer using this, so I've removed it